### PR TITLE
Remove settings added from system-tests

### PR DESCRIPTION
**Pull request by Sweetman**
### What does this PR do?
Remove integrated terminal settings
### What issues does this PR fix or reference?
Keep system-tests workspace clean

### CODE REVIEW

File: packages/system-tests/assets/sfdx-simple/.vscode/settings.json [hunk 1]
```diff
@@ -1,13 +1,5 @@
...
-},
-"terminal.integrated.env.linux": {
-    "SFDX_SET_CLIENT_IDS": "sfdx-vscode"
-},
-"terminal.integrated.env.windows": {
-    "SFDX_SET_CLIENT_IDS": "sfdx-vscode"
-},
-"eslint.nodePath": "/Users/nchen/Development/Node/salesforcedx-vscode/packages/salesforcedx-vscode-lwc/node_modules"
+  "eslint.nodePath":
+    "/Users/nchen/Development/Node/salesforcedx-vscode/packages/salesforcedx-vscode-lwc/node_modules"
```
Comment (Sweetman) [inline packages/system-tests/assets/sfdx-simple/.vscode/settings.json]: @vazexqi I'm assuming this should be hardcoded to your local env? Let me know if this should be changed to something else

Comment (vazexqi) [reply packages/system-tests/assets/sfdx-simple/.vscode/settings.json]: It's not just the entries. This whole file should be removed. That's why I was surprised when I saw that it hardcoded my entry and it was added to git.

Comment (vazexqi) [reply packages/system-tests/assets/sfdx-simple/.vscode/settings.json]: @Sweetman - Could you remove the file and let it run one more time through CI?

Comment (Sweetman) [reply packages/system-tests/assets/sfdx-simple/.vscode/settings.json]: @vazexqi sorry I was trying to remember if there was a reason we added this file before doing that. I think the reason we added the file for the autoUpdate field. If we do not include this and a branch is still on an earlier version than what's on the marketplace, then the tests will run with the marketplace version of the extensions. Do we want to keep this around for that reason? Just want to raise awareness of that but there should be no issue if the branch is always rebased to latest (which it needs to be anyway for the PR to be merged in).

Comment (vazexqi) [reply packages/system-tests/assets/sfdx-simple/.vscode/settings.json]: We only need `"extensions.autoUpdate": false` in there then. The rest can be removed.

**Comment by vazexqi** [comment]
@Sweetman - We cannot delete the entry for the eslint.nodePath yet. There is a sort of race condition going on with the way the tests are checking for it. So, the extension might not have finished bootstrapping yet before we do the check – this is why the AppVeyor tests are failing.

I need to think about this a bit more and see what condition we can latch on to get the test to work properly.

**Comment by vazexqi** [comment]
Closing in favor of #276.

**Pull request closed without merging**